Locale/LocaleEn.py

```diff
@@ -0,0 +1,60 @@
+# TODO translate bot to English
+from dataclasses import dataclass
+from Locale.LocaleBase import Locale
+
+
+@dataclass(frozen=True)
+class LocaleEN(Locale):
+    greet = ['Howdy',
+             '! Welcome to the InnoTyaga!\n'
+             'To display the available commands, type /help.\n'
+             'To display the all-time top participants, write /top.\n'
+             'To participate in the top, you need to register: /register.\n'
+             ]
+    schedule = "InnoTyaga classes are held at 19:00 on Wednesdays, Fridays and Sundays\n" \
+        "Wednesday - back and biceps, Friday - legs, Sunday - triceps and chest\n" \
+        "There are also occasional extra classes, information on which can be found in the club's main chat: @innotyaga"
```

Comment:
```suggestion
    schedule = 'Да похуй всем Серёжа Допизделся) Лучше выложи расклад карт таро на твою ЛЕГЕНДАРНЕЙШЕЮ "Прошедшей ночью данные ребята ломились не в свою квартиру в 130 доме. При вопросе "чонада" свалили в закат. Наверное дом перепутали. Или город. Короче скоро новый год и иронию судьбы косплеели ребята...надеюсь...\nГоворят местные перепутали 130 и 114 дом." в Иннополисе мы все ждем'  \
    'Info on the ever-changing class schedule subscribe to the official info channel @innotyaga'
```
As per @kupautist's request...